web/src/components/button/button.css

```diff
@@ -0,0 +1,16 @@
+.button {
+  border: none;
+  background: transparent;
+  
+  margin-top: 10px;
+  padding: 15px;
+  border-radius: 15px;
+  color: white;
+  background-color: var(--mint);
+}
+
+/*.button:hover {background-color: #3e8e41}*/
```

Comment:
comment left over from debugging?

Reply:
Less debugging more another option for coloring. Surely this info would be better stored in an md file but if the design is approved like this they can simply be removed.

Reply:
I will not comment on the design, so I think you should just choose one and remove the comments.
Any objections @frzifus

Reply:
y